django/thunderstore/community/context_processors.py

```diff
@@ -12,6 +13,11 @@ def community_site(request):
             "site_discord_url": request.community_site.community.discord_url or "",
             "site_wiki_url": request.community_site.community.wiki_url or "",
         }
+        result.update(
+            {
+                "site_favicon": f"{settings.STATIC_URL}favicon.ico",
```

Comment:
A couple of questions:
- Why is this change necessary / why isn't `site_icon` appropriate to use like it has been prior?
- If `site_icon` is no longer used, should it be removed?
- Does this need to be in the context processor at all, couldn't we just build the static path in the template?

Reply:
So this was an question that I forgot to ask previously; do any of the CommunitySites in production have icons? I'm guessing they don't. Because in the dev env, when I added icons (which I used as imagefields, before adding thumbnails for Community) for the CommunitySite, it would use that `icon` as the favicon. Which kinda confused me for a bit, before realizing that in the base.html template `site_icon` variable is used and that is defined on line 22  here in this file.

So I added `site_favicon` to decouple the icon field from favicon and fixed the base templates variable.

And yes if we have no interest in having custom favicons for a site, I recommend removing this part of the code.